src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -56,7 +56,6 @@
                 <Setter Property="Foreground"
                         Value="#bbbbbb" />
             </Style>
-            <pw:SelectedPythonEngineToMigrationConverter x:Key="SelectedPythonEngineToMigrationConverter" />
```

Comment:
SelectedPythonEngineToMigrationConverter class is not being used currently but could be useful in the future. Do we want to remove the class as well now?

Reply:
@reddyashish Can you remind us what does the converter do?

Reply:
Just a boolean converter to check if the engine version is Python2 https://github.com/DynamoDS/Dynamo/blob/2f9354254b83e9265d45a192a4e6e69c5d629eae/src/Libraries/PythonNodeModelsWpf/ValueConverter.cs#L8

Reply:
I would remove it